pages/docs/react/latest/styling.mdx

```diff
@@ -0,0 +1,207 @@
+
+- Open your VSCode settings
+- Search for `Tailwind CSS: Include Languages`
+- Add a new item with following settings:
+  - Item: `rescript`
+  - Value: `html`
+
+
+> **Hint:** `rescript-lang.org` actually uses TailwindCSS under the hood! Check out our [codebase](https://github.com/rescript-association/rescript-lang.org) to get some more inspiration on usage patterns.
+
+
+## CSS-in-JS
+
+Currently there are no official recommendations for CSS-in-JS yet due to the wildly different approaches on how to bind to CSS-in-JS libraries (going from simple to very advanced).
+
+The most minimalistic approach is to create simple bindings to e.g. [`emotion`](https://emotion.sh/docs/introduction) (as described [here](https://github.com/bloodyowl/rescript-react-starter-kit/blob/eca7055c59ba578b2d1994fc928d8f541a423e74/src/shared/Emotion.res)):
+
+```res
+// src/Emotion.res
+
+@module("@emotion/css") external css: {..} => string = "css"
+@module("@emotion/css") external rawCss: string => string = "css"
+@module("@emotion/css") external keyframes: {..} => string = "css"
+@module("@emotion/css") external cx: array<string> => string = "cx"
+
+@module("@emotion/css") external injectGlobal: string => unit = "injectGlobal"
+```
+
+This will give you straight-forward access to the `emotion` apis. Here's how you'd use them in your app code:
+
+```res
+let container = Emotion.css({
+  "color": "#fff",
+  "backgroundColor": "red"
+})
+
+let app = <div className={container} />
+```
+
+You can also use submodules to organize your styles more easily:
+
+```res
+module Styles = {
+  open Emotion
+  let container = css({
+    "color": "#fff",
+    "backgroundColor": "red"
+  })
+  // your other declarations
+}
+
+let app = <div className={Styles.container} />
+```
+
+Please note that this approach will not typecheck for invalid css attribute names. If you e.g. want to make sure that only valid CSS attributes are being passed, you could define your `css` function like this as well:
+
+```res
+@module("@emotion/css") external css: React.Style.t => string = "css"
+
+// Usage is slightly different (and probably less ergonomic)
+let container = ReactDOM.Style.make(~padding="20px", ())->css;
+
+let app = <div
+  className={container}
+/>
+```
+
+Here we used the already existing `React.Style.t` type to enforce valid CSS attribute names. 
+
+**Rule of thumb:** There's a spectrum on how type-safe an API might be, so choose a solution that fits to your team's needs.
+
+Last but not least, you can also bind to functions that let you use raw CSS directly:
+
+```res
+let container = Emotion.rawCss(`
+  color: #fff;
+  background-color: red;
+`)
+
+let app = <div className={container} />
+```
+
+For more CSS-in-JS ideas, you can also check out related discussions, or start a new topic on our [forum](https://forum.rescript-lang.org).
```

Comment:
Oh yeah you've got the link here too, just trying to make it more discoverable above.

Reply:
I think I will remove that sentence, since we already have it in the beginning!

Reply:
a wait, just realized this is for the CSS-in-JS part